### ISSUE TITLE
Windows 7 / 10 x64 install instructions

### ISSUE THREAD

**Issue by ZacDiggum** [cihologramas/pyoptools#46]
Hi there,

I tried to install pyoptools on my Win 10 x64 machine with Winpython 3.5 x64 via `pip install pyoptools`. Installation failed with a lot of error messages ending with:
`LINK : fatal error LNK1104: cannot open file 'kernel32.lib'`
Could you provide requirements, install instructions or maybe just some advice? :-)
I'd like to try out freecad-pyoptools when I get this working.

Thank you...

update:
after reinstalling Visual Studio and Windows 10 SDK I'm down to this error message:
`error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\cl.exe' failed with exit status 2`
and a lot of these:
`pyoptools\misc\Poly2D\Poly2D.c(18306): warning C4127: conditional expression is constant`
`pyoptools\misc\Poly2D\Poly2D.c(8855): warning C4204: nonstandard extension used: non-constant aggregate initializer`

**Comment by ijpulidos** [comment]
@ZacDiggum 
Windows support is still a work in progress, we don't really use windows to develop PyOpTools. 
I tried compiling pyoptools inside a WinPython command prompt terminal and I got a C2433 error, and lots of other warnings as you say.

I think the problem is that the Microsoft compiler doesn't support (probably is more strict in this sense) some inline modifiers, you can check https://groups.google.com/d/msg/cython-users/PtAXvswg9eg/pPEcVYxB0psJ and https://msdn.microsoft.com/en-us/library/9e7xa3fx.aspx for more info.

You could try compiling it via mingw, I haven't tried that yet because SourceForge is having problems with the service so I can't download mingw installers right now.

In conclusion, we don't have a working Windows build for pyoptools since we use linux for its development, but it is a work in progress.

**Comment by ijpulidos** [comment]
@ZacDiggum 
I managed to build pyoptools in windows using Cygwin and python2 (for some reason py3 didn't work, but it may have been a problem in my setup). The problem is that I'm having problems trying to run the examples, even just importing the OpenGL python module gives me problem, when I do a `from pyoptools.all import *` I get a "ImportError: ('Unable to laod OpenGL library', 'No such file or directory', 'OSMesa', None)"

If you want to test this setup in the following link there's a list of the packages I have installed in Cygwin https://gist.github.com/ijpulidos/4d2047d79814258254af1b6b941d77be 

Please let us know if you make some progress on this issue.

**Comment by ijpulidos** [comment]
OK, it was that libOSMesa8 and probably libOSMesa-devel packages were missing in my Cygwin install, so I just installed those and now pyoptools is working correctly. @ZacDiggum please check, I think this issue can be considered solved, though I think using Cygwin for pyoptools is not the ideal solution to have it running in windows, but it's the best option so far. I will update the install instructions for windows according to this information.

**Comment by ZacDiggum** [comment]
That's great news, thank you! I'll try to compile with Cygwin asap and close this issue.
